packages/util-waiter/src/waiter.ts

```diff
@@ -15,20 +15,18 @@ export interface WaiterConfiguration<Client> {
    * Abort controller. Used for ending the waiter early.
    */
   abortController?: AbortController;
-}
 
-export interface WaiterOptions<Client> extends WaiterConfiguration<Client> {
   /**
-   * The minimum amount of time to delay between retries in seconds. This value defaults
-   * to 2 if not specified. If specified, this value MUST be greater than or equal to 1
-   * and less than or equal to maxDelay.
+   * The minimum amount of time to delay between retries in seconds. This is the floor of the exponential backoff.
+   * This value defaults  to service default if not specified. If specified, this value MUST be greater than
+   * or equal to 1 and less than or equal to maxDelay.
    */
   minDelay?: number;
 
   /**
-   * The maximum amount of time to delay between retries in seconds. The maximum amount
-   * of time in seconds to delay between each retry. This value defaults to 120 if not
-   * specified (2 minutes). If specified, this value MUST be greater than or equal to 1.
+   * The maximum amount of time to delay between retries in seconds. This is the ceiling of the exponential backoff.
+   * The maximum amount of time in seconds to delay between each retry. This value defaults to service default
+   * if not specified. If specified, this value MUST be greater than or equal to 1.
```

Comment:
```suggestion
   * This value defaults to service default if not specified. If specified, this value MUST be greater than 
   * or equal to 1.
```
Remove duplicate sentence?

Reply:
Ahh nice catch. Done